assets/simple-boxes.js

```diff
@@ -725,40 +754,135 @@ $( document ).ready(function() {
     <div class="labelSelectorWrapper" data-handle-id="${handleId}" data-box-id="${boxId}"  style="top: ${labelY}; left: ${labelX}">
       <select class="labelSelector" data-handle-id="${handleId}" data-box-id="${boxId}">
         <option value="">Select a Label</option>
-`;
+`;    
 
-for(const allowedLabel of window.tagsList){
-  let selectedStr = allowedLabel == selectedLabel ? 'selected' : '';
 
-  labelStr+=`<option ${selectedStr} value="${allowedLabel}">${allowedLabel}</option>`
-}
 
+    for(const allowedLabel of window.tagsList){
+      let selectedStr = allowedLabel == selectedLabel ? 'selected' : '';
 
-labelStr+=`
+      labelStr+=`<option ${selectedStr} value="${allowedLabel}">${allowedLabel}</option>`
+    }
 
-      </select>
-    </div>
 
-    `;
-    $(labelStr).insertAfter('canvas#'+handleId);
+    labelStr+=`
+
+          </select>
+        </div>
+
+        `;
+        $(labelStr).insertAfter('canvas#'+handleId);
 
+      });
+
+  const saveAndClose = async (select) => {
+    let val = select.selectedOptions[0].value;
+    let boxId = select.getAttribute('data-box-id');
+    let handleId = select.getAttribute('data-handle-id');
+
+    // Save that selection
+    window.simpleBoxes._.handles[handleId].boxes[boxId].label = val;
+    let allBoxes = await window.simpleBoxes.getAllBoxes(handleId);
+
+    // Close it
+    select.parentNode.remove();
+    window.simpleBoxes._.handles[handleId].canvas.state.selectOpen = false;
+    } 
+
+    //When press down hot keys, select corresponding label options
+  document.addEventListener('keydown', async function(event) {
+    //hot key and key code pair
+    const pair = [];
+    //sct=10 is not merged yet, so hard code the hotkeys here
```

Comment:
i am guessing this means that this code should be updated at part of sct-10 then?  :+1: